pkg/relayer/miner/miner_test.go

```diff
@@ -1,10 +1,256 @@
 package miner_test
 
 import (
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"hash"
+	"sync"
 	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/pkg/observable"
+	"github.com/pokt-network/poktroll/pkg/observable/channel"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/miner"
+	"github.com/pokt-network/poktroll/pkg/relayer/protocol"
+	servicetypes "github.com/pokt-network/poktroll/x/service/types"
+)
+
+const testDifficulty = 2
+
+var (
```

Comment:
Should we add a comment that they can be regenerated via something like:

`go test -v -run TestFixttureGeneration_MineMockRelays ./pkg/relayer/miner/...`

Reply:
I refactored fixture generation into something a bit more formal that involves no copy/pasting to utilize.

Reply:
Great idea and a great pattern. 

Will start following this as well (and we'll add a note for us to document it)